### PR TITLE
Fix link to render textured meshes example

### PR DESCRIPTION
**Pull request by a-nau**
Great work! :)
Just found a link in the examples that is not working. This will fix it.

Best,
Alex

### CODE REVIEW

File: docs/notes/visualization.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ PyTorch3D provides a modular differentiable renderer, but for instances where we
 
 # Examples
 
-These rendering functions accept plotly x,y, and z axis arguments as `kwargs`, allowing us to customize the plots. Here are two plots with colored axes, a [Pointclouds plot](assets/plotly_pointclouds.png), a [batched Meshes plot in subplots](assets/plotly_meshes_batch.png), and a [batched Meshes plot with multiple traces](assets/plotly_meshes_trace.png). Refer to the [render textured meshes](../tutorials/render_textured_meshes.ipynb) and [render colored pointclouds](../tutorials/render_colored_points) tutorials for code examples.
```
[the remark this answers is not in the document]
Comment (a-nau) [reply docs/notes/visualization.md:13]: Thanks for the quick review!

Are you sure? At least for me clicking at [render textured meshes](https://pytorch3d.org/tutorials/render_textured_meshes.ipynb) from [visualizations](https://pytorch3d.org/docs/visualization) results in a 404, while the [render colored pointclouds](https://pytorch3d.org/tutorials/render_colored_points) link works fine.

However if I'm missing something, I can update the other link to include `ipynb` for sure.

[the remark this answers is not in the document]
Comment (patricklabatut) [reply docs/notes/visualization.md:13]: When visualized on GitHub, `render_colored_points` is the link leading to a "Page not found".

The confusion seems to come from browsing these notes on the dedicated PyTorch3D website instead, in which case the link to the `render textured meshes` is indeed broken and the proposed fix should work (but break on GitHub).

I would then suggest the following:
- Change the links to `render texture meshes` and `render color pointclouds` to be absolute links directly to the website pages, respectively `https://pytorch3d.org/tutorials/render_textured_meshes` and  `https://pytorch3d.org/tutorials/render_colored_points`
- Also fix another broken link above in the same file (the first one): `https://github.com/facebookresearch/pytorch3d/blob/main/pytorch3d/vis/plotly_vis.py`

**Comment by a-nau** [comment]
I updated all the links to absolute links as suggested. Thanks for your help and let me know if it's ok like this.

**Review by patricklabatut — APPROVED** [review]
Looks good.

**Pull request closed without merging**